### PR TITLE
Use tue-apt-select-mirror in CI

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

File: setup/tue-functions.bash [hunk 1]
```diff
@@ -32,8 +32,8 @@ function tue-apt-select-mirror
     # It uses apt-select to generate a new sources.list, based on the current one.
     # All Arguments to this functions are passed on to apt-select, so check the
     # apt-select documentation for all options.
-	hash pip2 2> /dev/null|| sudo apt-get install python-pip
-	hash apt-select 2> /dev/null|| pip2 install --user apt-select
+	hash pip2 2> /dev/null|| sudo apt-get install --assume-yes python-pip
+	hash apt-select 2> /dev/null|| sudo -H pip2 install apt-select
```
Comment (ar13pit) [inline setup/tue-functions.bash]: Why the change to a system-wide install?

Comment (MatthijsBurgh) [reply setup/tue-functions.bash]: With local install, I get a 127, command not found error.

Comment (ar13pit) [reply setup/tue-functions.bash]: That shouldn't happen. On my PC it works completely fine with local install

Comment (MatthijsBurgh) [reply setup/tue-functions.bash]: Yep also on my PC, but not in CI

Comment (ar13pit) [reply setup/tue-functions.bash]: So not in Docker you mean?

Comment (MatthijsBurgh) [reply setup/tue-functions.bash]: Yep, not in docker.
